applications/asset_tracker_v2/boards/thingy91_nrf9160_ns.overlay

```diff
@@ -20,7 +20,6 @@
 
 &spi3 {
 	adxl362: adxl362@0 {
-		wakeup-mode;
```

Comment:
Why is this removed?

Reply:
There is a problem with wake-up mode and setting the ODR (sampling rate) of the accelerometer.
This is to enable higher ODR I think.